src/makers/merryIoT/index.ts

```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 22,
```

Comment:
this `id` will need to be updated with your maker `id`. you're not guaranteed to get 22.